openhands-sdk/openhands/sdk/tool/schema.py

```diff
@@ -190,23 +193,87 @@ def visualize(self) -> Text:
 class Observation(Schema, ABC):
     """Base schema for output observation."""
 
+    content: list[TextContent | ImageContent] = Field(
+        default_factory=list,
+        description=(
+            "Content returned from the tool as a list of "
+            "TextContent/ImageContent objects. "
+            "When there is an error, it should be written in this field."
+        ),
+    )
+    is_error: bool = Field(
+        default=False, description="Whether the observation indicates an error"
+    )
+    error_message_header: str = Field(
+        default="Tool Execution Error. ",
+        description=(
+            "Header prepended to content and visualization when is_error is True"
+        ),
+    )
+
```

Comment:
Let's make this a class variable ERROR_MESSAGE_HEADER

and child class can override it